testing/unit-testing-jest/docs/testing-class.md

```diff
@@ -2,37 +2,28 @@
 To test a class, we need to instance it with the dependencies. Then, we can call methods and add assertions, for example:
 
 ```ts
-import { Injectable } from '@angular/core';
-import { LoggerService } from './logger.service';
-@Injectable({
-  providedIn: 'root',
-})
-export class CalculatorService {
-  constructor(private readonly logger: LoggerService) {}
```

Comment:
Do you think that we should add the testing process of an Angular service in another file?